### ISSUE TITLE
Hello! Where can I get '/dataset/annotations/crops.csv' ?

### ISSUE THREAD

**Issue by YaosenH** [3dperceptionlab/Visual-WetlandBirds#2]
Source code:
def get_dataloader(batch_size=32, shuffle=True, statistics = False, csv_path = '/dataset/annotations/crops.csv'):

    data_distribution = get_distribution()

    data = process_CSV(csv_path, data_distribution, statistics)
    
    train_dataset = Dataset(data['train_set']['frames'], data['train_set']['labels'])
    val_dataset = Dataset(data['val_set']['frames'], data['val_set']['labels'])
    test_dataset = Dataset(data['test_set']['frames'], data['test_set']['labels'])
    
    train_loader = DataLoader(train_dataset, batch_size=batch_size, shuffle=shuffle)
    val_loader = DataLoader(val_dataset, batch_size=batch_size, shuffle=False)
    test_loader = DataLoader(test_dataset, batch_size=batch_size, shuffle=False)
    
    return train_loader, val_loader, test_loader

**Comment by 3dplab** [comment]
Hi! The path you mentioned is the one we have chosen within the docker container we used during our experimentation. In order to get the code working you must replace the path provided by the one on which you have downloaded the data. In case you use Docker, ensure you have loaded the data in the container.
